templates/admin/controls/search-settings.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @var \Swiftype\SiteSearch\Wordpress\Admin\Page $this
+ */
+
+$engineSlug   = $this->getConfig()->getEngineSlug();
+$documentType = $this->getConfig()->getDocumentType();
+
+$searchSettingsRows = [
+  [
+    'title'        => __('Weights'),
+    'description'  => __("Your search relevance function determines the order of your search results."),
+    'url'          => sprintf("https://app.swiftype.com/engines/%s/document_types/%s/custom_queries/", $engineSlug, $documentType),
```

Comment:
I think all external links should open in a new tab, so that it is clearer to a user when they are in Wordpress and when they are on swiftype.com.